src/Layout/ConnectedLayout.tsx

```diff
@@ -66,7 +71,4 @@ const mapDispatchToProps = (dispatch: any): LayoutProps => ({
     },
 });
 
-export const ConnectedLayout = connect(
-    null,
-    mapDispatchToProps
-)(Layout as any);
+export const ConnectedLayout = connect(mapStateToProps, mapDispatchToProps)(Layout as any);
```

Comment:
Did we ever find out how to not need to resort to this hack, @Log234? Definitely something to think about for cleaning up the code.